tests/v2/validation/publicapi/crdgeneration/crd_generation.go

```diff
@@ -30,6 +30,7 @@ const (
 	crdJSONFilePath      = "../resources/crds.json"
 	roleTemplateJSONPath = "../resources/roleTemplate.json"
 	localCluster         = "local"
+	defaultBufferSize    = ""
```

Comment:
Just a thought, since defaultBufferSize is commonly used across both the tests that uses GetPodLogs, can we add this as a constant in the extension itself?